.gitignore

```diff
@@ -0,0 +1,2 @@
+main.exe
```

Comment:
Add bin/ directory to here. Also, there's no need to specifically write each executable you want to ignore - just write `*.exe`. By the way, there is plenty of predefined .gitignore files. You can just google something like "c++ gitignore" and just use it.